clerk/middleware.go

```diff
@@ -30,3 +31,21 @@ func addSessionToContext(request *http.Request, session *Session) *http.Request
 	updated := context.WithValue(ctx, ActiveSession, session)
 	return request.WithContext(updated)
 }
+
+func WithNetworkless(client Client) func(handler http.Handler) http.Handler {
```

Comment:
🔧 Perhaps, for consistency, `WithSessionNetworkless` is a better name?

Reply:
Not sure about the name yet, although this will be an easy change